tidy3d/plugins/adjoint/components/geometry.py

```diff
@@ -279,6 +288,15 @@ class JaxPolySlab(JaxGeometry, PolySlab, JaxObject):
         stores_jax_for="vertices",
     )
 
+    slab_bounds_jax: Tuple[JaxFloat, JaxFloat] = pd.Field(
```

Comment:
did this sneak in? although I think it's fine as we'll just do it in the slab_bounds PR

Reply:
I think I thought that it was required at some point because it's used in the polygon splitting [here](https://github.com/flexcompute/tidy3d/blob/06472617a646e6379b55c51b8afa2e626f36f9f3/tidy3d/plugins/adjoint/components/geometry.py#L850), although the PR would probably work fine without it. But this way, the `JaxComplexPolySlab` supports differentiating w.r.t. `slab_bounds` out of the box (tested) and the `slab_bounds` PR only needs to add the VJP.